dfastmi/io/DFastAnalysisConfigFileParser.py

```diff
@@ -0,0 +1,161 @@
+            Exception: When the given section or key does not exist.
+        """
+        return self._config_parser.getint(section, key, fallback=fallback)
+
+    def getfloat(self, section: str, key: str, fallback: float = 0.0) -> float:
+        """Retrieves a float value from the configuration.
+
+        Args:
+            section (str): The configuration section name.
+            key (str): The configuration key within the specified section.
+            fallback (float, optional): The fallback value if the key is not found. Defaults to 0.0.
+
+        Returns:
+            float: The float value of the configuration key.
+
+        Raises:
+            Exception: When the given section or key does not exist.
+        """
+        return self._config_parser.getfloat(section, key, fallback=fallback)
+
+    def getboolean(self, section: str, key: str, fallback: bool = False) -> bool:
+        """Retrieves a boolean value from the configuration.
+
+        Args:
+            section (str): The configuration section name.
+            key (str): The configuration key within the specified section.
+            fallback (bool, optional): The fallback value if the key is not found. Defaults to False.
+
+        Returns:
+            bool: The boolean value of the configuration key.
+
+        Raises:
+            Exception: When the given section or key does not exist.
+        """
+        return self._config_parser.getboolean(section, key, fallback=fallback)
+
+    def getstring(self, section: str, key: str, fallback: str = "") -> str:
+        """Retrieves a string value from the configuration.
+
+        Args:
+            section (str): The configuration section name.
+            key (str): The configuration key within the specified section.
+            fallback (str, optional): The fallback value if the key is not found. Defaults to an empty string.
+
+        Returns:
+            str: The string value of the configuration key.
+        """
+        return self._config_parser.get(section, key, fallback=fallback)
+
+    def get_range(
+        self, section: str, key: str, fallback: Tuple[float, float]
+    ) -> Tuple[float, float]:
+        """Retrieves a range (tuple of two floats) from the configuration.
+
+        The range is specified in the configuration file in the format "[x:y]", where
+        x and y are floats. This method parses that format and returns a tuple of two floats.
+
+        Args:
+            section (str): The configuration section name.
+            key (str): The configuration key within the specified section.
+            fallback (Tuple[float, float]): The fallback value if the key is not found or parsing fails.
+
+        Returns:
+            Tuple[float, float]: The range specified by the configuration key as a tuple of two floats.
+
+        Raises:
+            Exception: If parsing fails and no fallback is provided.
+        """
+        try:
+            ini_value = self.getstring(section, key, "")
```

Comment:
Maybe this can be splitup in a few smaller descriptive methods.

Reply:
discussed, out of scope.
Existing code which was moved.
Not to be updated in this issue anymore.